src/bag.cpp

```diff
@@ -7,7 +7,7 @@ bool Bag::s_seeded = false;
 
 Bag::Bag() : m_tetromino_sequence{} {
     if (!s_seeded) {
-        std::srand(std::time(nullptr));
+        std::srand((unsigned int) std::time(nullptr));
```

Comment:
Please no C-style casts :)
```suggestion
        std::srand(static_cast<unsigned int>(std::time(nullptr)));
```